core/src/main/scala/dx/core/Constants.scala

```diff
@@ -3,22 +3,28 @@ package dx.core
 import dx.api.ExecutionEnvironment
 
 object Constants {
-  // keys used in details of native applets
-  val ExecLinkInfo: String = "execLinkInfo"
-  val BlockPath: String = "blockPath"
-  val WfFragmentInputTypes: String = "fqnDictTypes"
-  val InstanceTypeDb: String = "instanceTypeDB"
+  // keys used in details of native applets, workflows
+  val ExecLinkInfo = "execLinkInfo"
+  val BlockPath = "blockPath"
+  val WfFragmentInputTypes = "fqnDictTypes"
+  val InstanceTypeDb = "instanceTypeDB"
+  val DxInstanceType = "dxInstanceType"
```

Comment:
I'd call this `staticInstanceType` to make it clear how it's different than just getting `instanceType` from the applet's runtimeExec.